src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCounter = 0;
+
+  const getSum = (num) => {
+    callCounter++;
+
+    return (operatar) => {
```

Comment:
There is a typo in the parameter name 'operatar'. It should be 'operator' or another appropriate name that correctly describes its purpose.